include/fastdds/dds/core/policy/QosPolicies.hpp

```diff
@@ -23,6 +23,7 @@
 #include <vector>
 
 #include <fastdds/dds/core/policy/ParameterTypes.hpp>
+#include <fastdds/dds/xtypes/type_representation/detail/dds_xtypes_typeobject.hpp>
```

Comment:
The idea is to include `fastdds/dds/xtypes/type_representation/TypeObject.hpp` that includes this header.

Reply:
Removed in [83756a6](https://github.com/eProsima/Fast-DDS/pull/4140/commits/83756a6445a363860a622ce24e1cbea2985a6c6a)

Reply:
That would create a circular dependency with dds_xtypes_typeobjectPubSubTypes.h
An ugly one :S